docs/content/.vitepress/config.js

```diff
@@ -24,42 +24,42 @@
 import {defineConfig} from 'vitepress'
 
 export default defineConfig({
-  title: 'Scaffolding',
-  description: 'Schematic library for Minestom',
+    title: 'Scaffolding',
+    description: 'Schematic library for Minestom',
 
-  themeConfig: {
-    sidebar: {
-      '/': getSidebar()
+    themeConfig: {
+        sidebar: {
+            '/': getSidebar()
+        }
```

Comment:
These lines should only be indented with 2 spaces.

Reply:
Thats correct how it is.